libs/browser-wallet-backend/src/lib/setup-listeners.js

```diff
@@ -1,186 +1,59 @@
 // import { createWindow } from '../src/windows.js';
+// import { disconnect, removeListener } from 'process';
```

Comment:
Delete?

Reply:
Yeah there will be much and more code that needs to be commented out and removed. I am focusing on getting it all working and then will go through and tidy up as we progress